packages/backend/database/src/drizzle/schema.ts

```diff
@@ -0,0 +1,454 @@
+    isVerified: boolean("is_verified").default(false),
+    createdAt: timestamp("created_at").defaultNow(),
+    updatedAt: timestamp("updated_at").defaultNow(),
+    tags: text("tags"),
+  },
+  (table) => ({
+    userEmailIdx: index("user_email_idx").on(table.email),
+    userLevelIdx: index("user_level_idx").on(table.levelId),
+    userCategoryIdx: index("user_category_idx").on(table.categoryId),
+  })
+);
+
+export const session = pgTable(
+  "session",
+  {
+    id: integer("id").primaryKey(),
+    createdAt: timestamp("created_at").defaultNow(),
+  },
+  (table) => ({
+    sessionCreatedAtIdx: index("session_created_at_idx").on(table.createdAt),
+  })
+);
+
+export const job = pgTable(
+  "job",
+  {
+    id: integer("id").primaryKey(),
+    title: varchar("title", { length: 255 }).notNull(),
+    levelId: integer("level_id").notNull(),
+    categoryId: integer("category_id").notNull(),
+    typeId: integer("type_id").notNull(),
+    location: varchar("location", { length: 255 }),
+    description: text("description"),
+    compensation: varchar("compensation", { length: 255 }),
+    applicationLink: varchar("application_link", { length: 255 }),
+    isExternal: boolean("is_external").default(false),
+    companyId: integer("company_id"),
+    createdAt: timestamp("created_at").defaultNow(),
+    updatedAt: timestamp("updated_at").defaultNow(),
+    tags: text("tags"),
+  },
+  (table) => ({
+    jobLevelIdx: index("job_level_idx").on(table.levelId),
+    jobCategoryIdx: index("job_category_idx").on(table.categoryId),
+    jobTypeIdx: index("job_type_idx").on(table.typeId),
+    jobCompanyIdx: index("job_company_idx").on(table.companyId),
+    jobCreatedAtIdx: index("job_created_at_idx").on(table.createdAt),
+  })
+);
+
+export const jobCategory = pgTable(
+  "job_category",
+  {
+    id: integer("id").primaryKey(),
+    title: varchar("title", { length: 255 }).notNull(),
+  },
+  (table) => ({
+    jobCategoryTitleIdx: uniqueIndex("job_category_title_idx").on(table.title),
+  })
+);
+
+export const jobLevel = pgTable(
+  "job_level",
+  {
+    id: integer("id").primaryKey(),
+    title: varchar("title", { length: 255 }).notNull(),
+  },
+  (table) => ({
+    jobLevelTitleIdx: uniqueIndex("job_level_title_idx").on(table.title),
+  })
+);
+
+export const jobType = pgTable(
+  "job_type",
+  {
+    id: integer("id").primaryKey(),
+    title: varchar("title", { length: 255 }).notNull(),
+  },
+  (table) => ({
+    jobTypeTitleIdx: uniqueIndex("job_type_title_idx").on(table.title),
+  })
+);
+
+export const jobSaved = pgTable(
+  "job_saved",
+  {
+    id: integer("id").primaryKey(),
+    jobId: integer("job_id").notNull(),
+    userId: integer("user_id"),
+    savedAt: timestamp("saved_at").defaultNow(),
+  },
+  (table) => ({
+    jobSavedJobIdx: index("job_saved_job_idx").on(table.jobId),
+    jobSavedUserIdx: index("job_saved_user_idx").on(table.userId),
+  })
+);
+
+export const jobViews = pgTable(

```

Comment:
Can we try to use drizzle's many to many feature here ? instead of creating a new table and have drizzle create it for us

Reply:
sorry but I didn't get this, if u may clarify it or send a link to check it I will be thankful.